File: tests/java/tests-java-gradle-junit5-allure3-selenide/src/test/java/tests/integration/HomeLayoutTests.java
package tests.integration;

import tests.TestBase;
import annotations.Layer;
import io.qameta.allure.Epic;
import io.qameta.allure.Feature;
import io.qameta.allure.Severity;
import io.qameta.allure.SeverityLevel;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;

@Layer("integration")
@Epic("Home")
@Feature("Home layout")
@Severity(SeverityLevel.NORMAL)
@DisplayName("Home layout mount")
class HomeLayoutTests extends TestBase {

    @Test
    @Tag("mount")
    @DisplayName("Home shows embedded header and reference layout")
    void homeLayoutIsMounted() {
        homePage.openPage()
                .shouldShowEmbeddedHeader()
                .shouldShowLayout();
    }
}

File: tests/csharp/tests-csharp-nunit-allure3-selenium/Tests/Ui/HomeLayoutTests.cs
using Allure.Net.Commons;
using Allure.NUnit.Attributes;
using Tests;

namespace Tests.Ui;

[AllureLabel("layer", "ui")]
[AllureEpic("Home")]
[AllureFeature("Home layout")]
[AllureSeverity(SeverityLevel.normal)]
[AllureSuite("Home layout mount")]
public sealed class HomeLayoutTests : TestBase
{
    [Test]
    [Category("ui")]
    [Category("mock")]
    [AllureName("Home shows embedded header and reference layout")]
    public void HomeLayoutIsMounted()
    {
        HomePage.OpenPage()
            .ShouldShowLayout()
            .Header.ShouldShowEmbeddedHeader();
    }
}
